src/index.js

```diff
@@ -0,0 +1,28 @@
+import apis from '../apis.json';
```

Comment:
I doubt that's a good idea... it won't be included in browserify will it?
that's why I made it a JS file...